PerformanceTests/Package.swift

```diff
@@ -0,0 +1,19 @@
+// swift-tools-version: 5.9
+
+import PackageDescription
+
+let package = Package(
+    name: "SDKWorkbench",
+    platforms: [.macOS(.v12), .iOS(.v15)],
+    dependencies: [
+        .package(name: "aws-sdk-swift", path: "../../aws-sdk-swift"),
+    ],
+    targets: [
+        .executableTarget(
+            name: "PerformanceTestScript",
```

Comment:
I think `PerformanceTestRunner` might be a better module name that fits same naming pattern used for smoke tests + it's more indicative that it's a Swift executable rather than a normal script file like `test.sh`.